service/src/main/java/greencity/service/notification/AbstractNotificationProvider.java

```diff
@@ -9,14 +9,17 @@
 import greencity.enums.NotificationReceiverType;
 import greencity.exceptions.user.UserNotFoundException;
 import greencity.repository.NotificationTemplateRepository;
+import lombok.Getter;
 import lombok.RequiredArgsConstructor;
 
 import javax.annotation.PostConstruct;
 
 @RequiredArgsConstructor
+@Getter
```

Comment:
Why you added this?

Reply:
I added it to call provider.getNotificationType() to pass notification type of provider into sendNotification function